OpenShift/helm/nexus-iq/templates/ingress.yaml

```diff
@@ -26,22 +28,18 @@ spec:
     {{- end }}
   {{- end }}
   rules:
-  {{- range .Values.ingress.hosts }}
-    - host: {{ . | quote }}
+    - host: iq-server.demo
       http:
         paths:
           - path: {{ $ingressPath }}
             backend:
               serviceName: {{ $fullName }}
-              servicePort: http
-  {{- end }}
-  {{- range .Values.service.additionalPorts }}
-    - host: {{ .host | quote }}
+              servicePort: 8070
+    - host: admin.iq-server.demo
```

Comment:
We've obviously not used the ingress nodes yet, so I know almost nothing about it. Is this OK to be hardcoded like this?

Reply:
No, another 'cleanup' item. I didn't want to solve how to write a template just yet and focused on how to get the deployment to work with a service and an optional ingress. I kept tripping on doing this pattern for more than one port so resorted to the hard code and at some point even removing the second port for now.